frontend/js/utils/WebAPIUtils.js

```diff
@@ -140,6 +143,10 @@ var APIUtils = {
     POST('/api/games/'+gameID+'/actions/leave', {gameAuth:gameAuth}, success, error);
   },
 
+  forfeitGame: function(gameID, gameAuth, success, error) {
+    POST('/games/'+gameID+'/actions/resign', {gameAuth:gameAuth}, success, error);
```

Comment:
Doesn't currently work? Should be '/api/games/' in the front, rather than just '/games/'.

Also, do you prefer the term forfeit over resign? We should probably stick with "resign", because at least on the old site, "forfeit" is reserved for things like game loss due to no-show in a pre-scheduled event game. I think I've seen the this terminology difference of "forfeit" (loss by failing to show, or by default, or by violating a policy, or by misconduct) vs "resign" (loss by voluntarily conceding the win during the game) in other competitive game/sports contexts besides Arimaa.